### ISSUE TITLE
Preserving existing annotations in stubgen

### ISSUE THREAD

**Issue by JukkaL** [python/mypy#2106]
If code has annotations, it would be nice if stubgen would preserve them instead of filtering them out. Example (input -> output):

```
from mod import Thing
import mod2

def f(x: Thing) -> int:
    mod2.do_stuff()
    # more implementation
```

->

```
from mod import Thing

def f(x: Thing) -> int: ...
```

Maybe we should also (optionally) run type inference so that we could output inferred types for module-level variables and attributes.

See also discussion in #1899.

**Comment by yoongu** [comment]
It would be super helpful to us if this got implemented!

**Comment by ilevkivskyi** [comment]
This is now implemented in #3169.
